components/dropdown/Dropdown.js

```diff
@@ -44,16 +59,27 @@ const Dropdown = ({ isOpen, children, className, content, autoClose, autoCloseOu
         };
     }, []);
 
+    const alignClass = ALIGN_CLASSES[align];
+    const dropdownClassName = `dropDown ${alignClass ? alignClass : ''} pm-button`;
+    const contentClassName = `dropDown-content ${narrow ? 'dropDown-content--narrow' : ''}`;
+    const caretContent = caret && <Icon className="expand-caret" size={12} name="caret" />;
+
     return (
-        <div className="dropDown" ref={wrapperRef}>
-            <Button className={className} onClick={handleClick} aria-expanded={open}>
-                {content}
-            </Button>
-            {open ? (
-                <div className="dropDown-content" onClick={handleClickContent}>
-                    {children}
-                </div>
-            ) : null}
+        <div className={`${dropdownClassName} ${className}`} ref={wrapperRef}>
+            <button
```

Comment:
I suppose you have a reason to not use the `Button` component here. Then did you check how make the dropdown looks primary if needed?